lib/posix-sysinfo/sysinfo.c

```diff
@@ -98,6 +103,13 @@ long sysconf(int name)
 	if (name == _SC_PAGESIZE)
 		return __PAGE_SIZE;
 
+		if (name == _SC_PHYS_PAGES){
+		struct uk_pagetable *pt = ukplat_pt_get_active();
+		__sz total_memory = pt->fa->total_memory;
+		__sz total_pages = total_memory/__PAGE_SIZE;
+		return total_pages;
+	}
```

Comment:
Please fix indentation for the block and spacing around `/` and before the `{`. Also you need to enclose this block with an `#ifdef` since the functionality you use is not available without paging support. Generally, we also put all variables at the top. However, we recently decided that it would be ok to define variables at the top of a new scope. So this is fine. Nevertheless, add a newline in this case.

```suggestion
#ifdef CONFIG_HAVE_PAGING
	if (name == _SC_PHYS_PAGES) {
		struct uk_pagetable *pt = ukplat_pt_get_active();
		__sz total_memory = pt->fa->total_memory;
		__sz total_pages = total_memory / PAGE_SIZE;
		
		return total_pages;
	}
#endif /* CONFIG_HAVE_PAGING */
```